config/rewrites.d/support.rackspace.com.json

```diff
@@ -965,6 +965,13 @@
         "to": "/how-to/enable-email-archiving-for-office-365/",
         "rewrite": false,
         "status": 301
+      },
+      {
+        "description": "redirect Suported Operating Systems in Rackspace Public Cloud to new /how-to/supported-operating-systems-and-browsers-in-rackspace-public-cloud/",
```

Comment:
Add another "p" to "Suported." Will merge this after the changes to the other file have been merged.